src/util/apiUtils.js

```diff
@@ -201,56 +223,79 @@ export const apiResponseToQueryResponse = query => ({ value, meta }) => ({
 	}
 });
 
-/**
- * Parse request for queries and request options
- * @return {Object} { queries, externalRequestOpts }
- */
-export function parseRequest(request, baseUrl) {
-	const {
-		headers,
-		method,
-		query,
-		payload,
-		state,
-	} = request;
+export function getAuthHeaders({ state }) {
+	if (!state.MEETUP_MEMBER && state.oauth_token) {
+		return {
+			authorization: `Bearer ${state.oauth_token}`,
+		};
+	}
+	const cookies = { ...state };
+	const csrf = uuid.v4();
+	cookies.MEETUP_CSRF = csrf;
+	cookies.MEETUP_CSRF_DEV = csrf;
```

Comment:
Can we document how the CSRF token / api exchange works? I've already forgotten the conversation we had with Doug awhile back.